locales/en/actions.json

```diff
@@ -6,6 +6,7 @@
   "previous_week": "Previous week",
   "next_week": "Next week",
   "current_week": "This week",
+  "date_select": "Jump to date",
```

Comment:
Disagree

` "current_day": "Today"`

Reply:
That’s why I said “most” 😉

Reply:
It does feel like you’re basing your addition on the 5% instead of the 95% though. All the other submit action keys follow the pattern of using the value in the key.